modules/test/tls/python/src/tls_module.py

```diff
@@ -280,7 +280,7 @@ def _validate_tls_client(self, client_ip, tls_version):
 
     # Generate results based on the state
     result_message = 'No outbound connections were found.'
-    result_state = None
+    result_state = 'Feature Not Present'

```

Comment:
Replacing the None with a feature not present is not correct. There is no way to determine if a device supports TLS client feature if not communicating during test, this needs further configuration.